speechmatics/models.py

```diff
@@ -34,6 +34,9 @@ class TranscriptionConfig:
     max_delay: float = None
     """Maximum acceptable delay."""
 
+    max_delay_mode: str = None
+    """Indicates if we will use max_delay or flexible enpointing."""
```

Comment:
This isn't quite accurate I think - in both cases we use max_delay, it this just controls what made we use it in. In one case we strictly adhere to it (fixed), in the other we allow a bit longer segments in places where we know adhering to it would impact the accuracy significantly.

Reply:
Changed it to reflect accurate instructions.